### DISCUSSION TITLE
Cannot access Document linked

### DISCUSSION THREAD

**Discussion by hiamitabha** [hickeng/financial#145] [General]
>There's a [more in-depth but still quick overview](https://docs.google.com/document/d/1VdKL3F3ne8Spj03KGFWqioh-sFGV7BTDGLDOfXrgY0I/edit?usp=sharing) of how VMW shares convert to AVGO+cash, provided by Dan Kan. You don't need to know this, but if you want to understand the basics behind the numbers the sheet generates, this is a good starting point.

I am unable to access this document... thought it will be an interesting read.
The error I get is: "The organization that owns this item doesn't allow you to access it"

**Comment by hickeng** [comment]
Thanks for the heads up! I've asked Dan for a version shared via personal ID.

**Comment by hickeng** [comment]
I've updated the link with a new one from Dan that should be globally accessible.
